CONTRIBUTING.md

```diff
@@ -10,25 +10,29 @@ For developers interested in expanding `pymatgen` for their own purposes, we rec
 
 1. Create a free GitHub account (if you don't already have one) and perform the necessary setup (e.g., install SSH keys etc.).
 
-1. Fork the `pymatgen` GitHub repo, i.e., go to the main [`pymatgen` GitHub repo](https://github.com/materialsproject/pymatgen) and click fork to create a copy of the `pymatgen` code base on your own GitHub account.
+2. Fork the `pymatgen` GitHub repo, i.e., go to the main [`pymatgen` GitHub repo](https://github.com/materialsproject/pymatgen) and click fork to create a copy of the `pymatgen` code to your own GitHub account.
 
-1. Install `git` on your local machine (if you don't already have it).
+3. Install `git` on your local machine (if you haven't already).
 
-1. Clone *your forked repo* to your local machine. You will work mostly with your local repo and only publish changes when they are ready to be merged:
+4. Clone *your forked repo* to your local machine. You will work mostly with your local repo and only publish changes when they are ready to be merged:
 
     ```sh
     git clone https://github.com/<username>/pymatgen
+
+    # (Alternative/Much Faster) If you don't need a full commit history/other branches
+    # git clone --depth 1 https://github.com/<username>/pymatgen
```

Comment:
I think it might be good to provide this alternative as I think most new contributors don't really need the complete commit history or other branches, they would just branch from main.

To clone the complete repo:
```
git clone https://github.com/materialsproject/pymatgen.git  48.11s user 15.68s system 69% cpu 1:31.62 total
```

Depth one clone:
```
git clone https://github.com/materialsproject/pymatgen.git --depth 1  7.60s user 2.75s system 31% cpu 32.622 total
```